odoo/tests/common.py

```diff
@@ -708,6 +711,7 @@ class TransactionCase(BaseCase):
     registry: Registry = None
     env: api.Environment = None
     cr: Cursor = None
+    rollback_sequences = False
```

Comment:
I am really not fond of that feature, which feels too brittle to me.  Add a section in the docstring to explain parameter `rollback_sequences` and discourages its use in general :wink: